apps/condo/domains/meter/schema/MeterReading.test.js

```diff
@@ -185,6 +186,81 @@ describe('MeterReading', () => {
             })
         })
 
+        test('employee with division limited role: can read meters only from his divisions', async () => {
```

Comment:
Please, introduce a `describe` blocks for employee with and without division scoped visibility.

For example:

```
describe('employee with division scoped visibility', () => {
  it('can read meters only from his divisions')
})

describe('employee without division scoped visibility', () => {
  it('can read all …')
})
```